modules/postgres/options.go

```diff
@@ -0,0 +1,14 @@
+package postgres
+
+type SSLVerificationMode string
+
+type SSLSettings struct {
+	// Path to the CA certificate file
+	CACertFile string
+	// Path to the client certificate file
+	CertFile string
+	// Path to the key file
+	KeyFile string
+	// Entrypoint used to override and set SSL ownership
+	Entrypoint string
```

Comment:
I'd not mix the SSL Settings with an entrypoint to change ownership. I instead would pass another functional option to add a file, separating concerns. Wdyt? Doing that will allow to, at some point, extract the SSL capabilities and add them to the core, so any module can benefit from it (We have some modules already doing it).

Reply:
No problem. I was thinking the same thing this morning.

In some ways I don't like exposing the `Entrypoint` because it's such a large escape hatch.